src/main/java/mate/dao/DriverDaoImpl.java

```diff
@@ -73,14 +75,16 @@ public Driver create(Driver driver) {
     @Override
     public Driver update(Driver driver) {
         String query = "UPDATE drivers "
-                + "SET name = ?, license_number = ? "
+                + "SET name = ?, license_number = ?, login = ?, password = ? "
```

Comment:
Do we need to update our login and password?

Reply:
Not sure but why not? In future I will add changepassword controller for example.